src/store/csv/stub/store.py

```diff
@@ -58,3 +93,26 @@ def get_version(self, dataset_id: str, edition_id: str, version_id: str, request
         )
 
         return FileResponse(csv_path) if csv_path else None
+
+
+    def get_latest_version(self, csv_identifier: str) -> Optional[str]:
```

Comment:
there's a nuance here I probably didn't capture on the card.

- if someone requests a csv at the dataset level we return the latest verion of the **latest** edition.
- if someone requests a csv at the edition level we return the latest version of **that** edition.

haven't dug into this code properly so if its already doing that then ignore this comment, if its not can you change it to do so please.